internal/spokes/spokes.go

```diff
@@ -116,23 +116,25 @@ func (r *SpokesReceivePack) Execute(ctx context.Context) error {
 		// * If we found a general check-connectivity error, let's check every individual command
 		// * If no individual error has been found and the reportStatusFF settings is true, let's see if the reference update could be a fast-forward
 		for i := range commands {
-			var singleObjectErr error
 			c := &commands[i]
-			c.reportFF = "ok"
-			if err != nil {
-				singleObjectErr = r.performCheckConnectivityOnObject(ctx, c.newOID)
-				if singleObjectErr != nil {
-					c.err = fmt.Sprintf("missing required objects: %s", err.Error())
-					c.reportFF = "ng"
+			if !c.rejected {
```

Comment:
This is getting a bit hard to read.  Would it be better to switch this around a bit, and have a short circuit of the loop such as 

```
if c.rejected {
    continue
}
if !err {
[...]
}
```


I think that would make this easier to read.

Reply:
And another thought, instead of having the `c.rejected` flag here would it be easier to just check `c.err != ""`?  It took me a little bit to understand why we had this separate `rejected` flag and what it indicates.

Reply:
Make it easier to read: 5fcf5b7d109ed07bb58a68814e0d6b1ff7da2bc6

Regarding the rejected field, I have changed it here cdf1e49f9403870b8f65f4558bda423fddc11b61 For me, it's clear with the original approach, but I have no problem reusing the error field